platform/server-bundle/src/main/resources/webapp/js/controllers.js

```diff
@@ -396,6 +396,7 @@
                 $scope.i18n = data;
             }),
             $scope.doAJAXHttpRequest('GET', 'lang/list', function (data) {
+                if(data['zh_TW.Big5'] !== undefined) { delete data['zh_TW.Big5']; }
```

Comment:
Let's style this code properly. The content of the if, as well as the closing brace should be in a new line. Moreover, let's add a comment with the ticket number, explaining that we temporarily hide this, until it's fixed.